3 Fixes + 2 Bridges (Rumble & Tysol)

sudwebdesign opened the pull request:
Fix Notice: Undefined index: format
Fix Warnings: Se Coucher Moins Bête
[HentaiHavenBridge] new uri
[TysolFrBridge] Add bridge
[RumbleBridge] Add bridge

sudwebdesign committed:
Fix Notice: Undefined index: format

Found with
?action=display&bridge=Demo&context=testNumber&testNumber=2

sudwebdesign committed:
Fix Warnings: Se Coucher Moins Bête

Add: remove img "anecdote-premium-blurry.jpg"
Fix: Warnings Creating default object from empty value

sudwebdesign committed:
[HentaiHavenBridge] new uri

.org in error

sudwebdesign committed:
[TysolFrBridge] Add bridge

sudwebdesign committed:
[RumbleBridge] Add bridge

em92 commented:
Hi, @sudwebdesign! Please make separate PR for each bridge. Also make sure you fixed linting errors

sudwebdesign committed:
Fix bridges linting errors

sudwebdesign committed:
Fix require in list type + indents

sudwebdesign committed:
Fix Line exceeds maximum limit of 120 characters

sudwebdesign committed:
Fix Expected 1 space after ":"; newline found

sudwebdesign committed:
Fix Expected 1 space before ":"; newline found

sudwebdesign committed:
Fix paradox of max (120) chars per line

27 | ERROR | [x] Expected 1 space after ":"; newline found
27 | ERROR | [x] Whitespace found at end of line

sudwebdesign commented:
Hi @em92 all marks is green now ;-)

sudwebdesign committed:
readme

Bockiii committed:
Merge branch 'master' into master

Bockiii commented:
Hi,

I removed your contributions that caused the merge conflicts as both hentaiheaven and scmb were updated since this pr was made.

Please check if your changes still make sense. Also check if the change to the DisplayAction is really needed.

The rumble bridge needs example/default values that actually produce an output.

The tysol bridge is not working at all. tysol.fr leads to tysol.pl when I open it in the browser. Bridge obsolete?

Please update all the things. Thanks!

dvikan commented:
Please make separate PRs for each bridge.
